docs/kind.md

```diff
@@ -117,11 +122,6 @@ kind create cluster --config kind-config.yml
 ### Deploy Antrea to your Kind cluster
 
 ```bash
-# pull the Antrea Docker image
-docker pull projects.registry.vmware.com/antrea/antrea-ubuntu:<TAG>
-# load the Antrea Docker image in the Nodes
-kind load docker-image projects.registry.vmware.com/antrea/antrea-ubuntu:<TAG>
-# deploy Antrea
 kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea.yml
```

Comment:
I am wondering why are these removed instead of updating them?

Reply:
They are not necessary, I don't know why they were added but I think there is no reason to ask users to do it via 3 instructions when it can be done via 1 instruction. Especially when they need to edit the <TAG> placeholder.

The instructions starting at L61 are different because they are describing how to pre-load the images. For normal deploying, we should keep things simple.